pkg/storkctl/clusterdomainupdate_test.go

```diff
@@ -0,0 +1,223 @@
+// +build unittest
+
+package storkctl
+
+import (
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/stretchr/testify/require"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func createClusterDomainUpdate(t *testing.T, name string, clusterDomain string, active bool) *storkv1.ClusterDomainUpdate {
+	cdu := &storkv1.ClusterDomainUpdate{
+		ObjectMeta: meta.ObjectMeta{
+			Name:      name,
+			Namespace: "kube-system",
+		},
+		Spec: storkv1.ClusterDomainUpdateSpec{
+			ClusterDomain: clusterDomain,
+			Active:        active,
+		},
+	}
+	_, err := k8s.Instance().CreateClusterDomainUpdate(cdu)
```

Comment:
Use storkctl to create to test that codepath.

See here for example: https://github.com/libopenstorage/stork/blob/master/pkg/storkctl/migration_test.go#L26

Reply:
I didn't add a storkctrl create command for clusterdomainupdate. Isn't activate/deactivate adequate for updating the cluster domains.

Reply:
Ah use the activate/deactivate command then?

Reply:
Yes will do